constants/organizationMembers.js

```diff
@@ -1,40 +1,32 @@
 export default {
+  eternoendless: {},
   fabienvallon: {},
-  MatShir: {},
-
-  sowbiba: {},
   jolelievre: {},
-  kpodemski: {},
+  MatShir: {},
   matks: {},
+  'Robin-Fischer-PS': {},
+  Hlavtox: {},
+  'jf-viguier': {},
+  kpodemski: {},
+  RosaBenouamer: {},
+  Zoefinders: {},
+  ElodieOS: {},
+  TristanLDD: {},
+  'l-delin': {},
+  sowbiba: {},
   'matthieu-rolland': {},
-  eternoendless: {},
   PululuK: {},
   atomiix: {},
   NeOMakinG: {},
-
-  Hlavtox: {},
-
-  zuk3975: {},
```

Comment:
he's still part of the project

Reply:
OK, but @Progi1984 told me that committers shouldn't be listed here.

Reply:
don't they? why not?

Reply:
I don't know what are the rules for this configuration. I thought it was the users on https://www.prestashop-project.org/project-organization/people-and-roles/ ...